README.md

```diff
@@ -27,6 +30,39 @@ Import stylesheets and require javascripts:
 //= require active_admin/searchable_select
 ```
 
+##### Using assets via Webpacker (or any other assets bundler) as a NPM module (Yarn package)
+
+Execute:
+
+    $ npm i @codevise/activeadmin-searchable_select
+
+Or
+
+    $ yarn add @codevise/activeadmin-searchable_select
+
```

Comment:
Should we encourage pinning a specific version here to keep things in sync with the gem?

Reply:
Usually the package version sync (bumped) together with gem version, but this is not necessary unless we change some javascript, stylesheets or readme in gem. By default when we call `yarn add @codevise/activeadmin-searchable_select` added the latest published version, if we need to specific version we just call `yarn add @codevise/activeadmin-searchable_select@1.6.0` .